lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -4,6 +4,19 @@
 
 module KubernetesDeploy
   class KubernetesResource
+    def self.logger=(value)
+      @logger = value
+    end
+
+    def self.logger
+      @logger ||= begin
+        l = Logger.new($stderr)
+        l.formatter = proc do |_severity, _datetime, _progname, msg|
+          "#{msg}\n"
```

Comment:
I'm not convinced they'd add value rather than noise. The kube status logs happen at a defined interval, and the others are relaying logs from somewhere else, which already have their own timestamps ... imagine having another timestamp on the 2nd-to-bottom line in the screenshot. Wouldn't that just be confusing? I'd be more in favour of adding timestamps to the regular logger (coloured lines in the screenshot).